src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -36,10 +37,12 @@ public FindCommand(Set<String> keywords) {
 
     @Override
     public CommandResult execute() {
-        final List<ReadOnlyPerson> personsFound = getPersonsWithNameContainingAnyKeyword(keywords);
+        Set<String> lowerCaseKeyword = Utils.keywordToLowerCaseSet(keywords);
+        final List<ReadOnlyPerson> personsFound = getPersonsWithNameContainingAnyKeyword(lowerCaseKeyword);
```

Comment:
Be careful here, as I don't know that you convert the `name` into lowercase without checking the method.